app/model/user.js

```diff
@@ -209,6 +209,10 @@ userSchema.pre('save', function (next) {
 
 // sync with LDAP
 userSchema.pre('save', function (next) {
+  // disable ldap for dev
+  if (!process.env.NODE_ENV && !process.env.LDAP) {
```

Comment:
This is not correct. NODE_ENV=production is valid -- I shouldn't have to specify BOTH -- either/or